scripts/installer/install

```diff
@@ -829,8 +829,8 @@ else
     print "\nCreating Postgres database:\n";
     
     my $psql_command =
-	$psql_admin_exec
-	. "/createdb -h " . $CONFIG_DEFAULTS{'POSTGRES_SERVER'} . " -U postgres "
+	$psql_exec
+	. "/createdb -h " . $CONFIG_DEFAULTS{'POSTGRES_SERVER'} . " -U dvnapp "
```

Comment:
I'm just parroting something that @landreev said yesterday but we don't want "dvnapp" hard-coded here. It should come from `$CONFIG_DEFAULTS{'POSTGRES_USER'}`. See https://github.com/IQSS/dataverse/blob/v4.5/scripts/deploy/phoenix.dataverse.org/install#L8 for an example of where we use `export DB_USER=dvnapp` in a shell script to set this environment variable before calling the `install` script. We want to make sure this is configurable and not hard-coded.